### PR TITLE
Support kirkstone and use new override

### PR DESCRIPTION
**Pull request by alistair23**
Update to use the new override syntax and support the kirkstone release

### CODE REVIEW

**Comment by alistair23** [comment]
@csonsino

File: recipes-bsp/trusted-firmware-a/trusted-firmware-a_%.bbappend [hunk 1]
```diff
@@ -1,3 +1,3 @@
-COMPATIBLE_MACHINE_append_a64 = "|a64"
-COMPATIBLE_MACHINE_append_h6 = "|h6"
+COMPATIBLE_MACHINE:append_a64 = "|a64"
+COMPATIBLE_MACHINE:append_h6 = "|h6"
```
[the remark this answers is not in the document]
Comment (alistair23) [reply recipes-bsp/trusted-firmware-a/trusted-firmware-a_%.bbappend]: Good point, fixed

File: conf/layer.conf [hunk 2]
```diff
@@ -9,4 +9,4 @@ BBFILE_COLLECTIONS += "pine64-layer"
 BBFILE_PATTERN_pine64-layer = "^${LAYERDIR}/"
 BBFILE_PRIORITY_pine64-layer = "5"
 
-LAYERSERIES_COMPAT_pine64-layer = "dunfell gatesgarth hardknott"
```
[the remark this answers is not in the document]
Comment (alistair23) [reply conf/layer.conf:12]: I'm happy to add support, it's just hard to test them all. I have added `honister` here as well

**Review by csonsino — APPROVED** [review]
Looks good to me

**Pull request merged**